src/olympia/reviewers/views.py

```diff
@@ -1201,8 +1202,9 @@ def download_git_stored_file(request, version_id, filename):
         raise http.Http404()
 
     actual_blob = serializer.git_repo[blob_or_tree.oid]
+
     response = http.FileResponse(
-        memoryview(actual_blob),
+        io.BytesIO(memoryview(actual_blob)),
```

Comment:
yeah it seems like there should be a more direct way without having to wrap `serializer.git_repo[blob_or_tree.oid]` in both a `memoryview` and `io.BytesIO` but finding out would just be trial and error and it seems you've done that.